tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -666,7 +668,7 @@ class ScopedActivationDescriptor {
         mode = CUDNN_ACTIVATION_TANH;
         break;
       default:
-        LOG(ERROR) << "unrecognized activation mode: "
+        LOG(FATAL) << "unrecognized activation mode: "
```

Comment:
Let's revert this -- we don't want to crash the program -- we'd prefer to return a bad status message.